File: C++/Armstrong_no.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

#ifndef ONLINE_JUDGE
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
#endif
	// start code from here
	//cout << "Hello";
	int n;
	cin >> n;
	int temp = n;
	int sum = 0;
	while (n > 0) {
		int r = n % 10;
		sum = sum + (r * r * r);
		n = n / 10;
	}
	if (sum == temp) {
		cout << temp << " Number is Armstrong Number";
	}
	else {
		cout << "not a armstrong number ";
	}


	return 0;
}

File: Javascript/armstrong_no.js

// Program to chech whether is a number is armstrong number or not in javascript

function armstrong(num){

    var temp=num;
    var arm = 0;
    while(temp>0)
    {
        var a=temp%10;
        temp=parseInt(temp/10);
        arm=arm+a*a*a;
    }

    if(arm == num)return true;

    return false;

}

console.log(armstrong(370));

//Output
// true